Sources/TuistKit/Services/ImportFinderService.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+import Path
+import ProjectAutomation
+import Tools
+import TuistCore
+import TuistLoader
+import TuistSupport
+import XcodeGraph
+
+enum ImportFinderServiceError: FatalError, Equatable {
+    case targetNotFound(target: String)
+
+    var description: String {
+        switch self {
+        case let .targetNotFound(target):
+            "Target \(target) was not found"
+        }
+    }
+
+    var type: ErrorType {
+        switch self {
+        case .targetNotFound:
+            .abort
+        }
+    }
+}
+
+final class ImportFinderService {
```

Comment:
I'd name this one `GraphImplicitImportLintService` and make it more generic to work with the entire graph. What do you think? I think it's more representative of what the service does.

Reply:
Done